### ISSUE TITLE
Corpus composition: titles per author

### ISSUE THREAD

**Issue by lb42** [distantreading/WG1#19]
An additional constraint on corpus composition is that there should not be more than three titles for any given author, and that no more than ten authors in the whole collection should be represented by more than one title (see [reminder](http://lists.distant-reading.net/mailman/private/wg1/2018-October/000105.html) )
The existing collections don't all respect these maxima -- the Portugese collection for example has many authors with way more than three titles; the French collection has way more than ten authors with three. Are these numbers meant to be maxima or minima? My (strong) recommendation is that they should be maxima. Even at the level proposed they risk unbalancing the diversity of the corpus.

**Comment by michaelprem** [comment]
I am thinking that this constraint may be difficult to live up to in the phase when the collection is still under construction.
May be a policy of building and thereafter pruning (provided that the process is automatized) is a better way to work? 
Do we have a definition of diversity?
Could two works of the same author twenty years apart also represent diversity?                

Sorry I cannot attend Lisboa, BTW, wish you a wonderful meeting

**Comment by lb42** [comment]
All the constraints about corpus composition relate to the final 100 text corpus, so yes there is ample opportunity to  include/exclude texts as we go. But I don't think this can be automated, certainly not without clearer criteria.

**Comment by lb42** [comment]
Not "at least 10" but "9 to 11" authors and exactly three titles for each. This is difficult or impossible for some languages.

**Comment by luhpeg** [comment]
Christoph's reminder (October 2018) and the selection criteria only speak of at the minimum number of that representation.  We invested a lot of human power to make the list of works and the encoding ready. I personally would be quite disappointed if the works we carefully selected and partly encoded following the given criteria would be moved to folder which is not part of the actual core activity of the project.

Concerning the balance: we tried to follow the literary canon represented in renown handbooks of literary history. Those authors that should be included  - if we follow the new decision of maximum 3 works per author -  are not present in the literary canon. Most likely a lot of low quality texts would then be included that nobody cared even in the 19th Century instead of works that have considerable influence on Hungarian literature.

Apart from the fact that following the new criteria would take a lot of time it seems to be impossible to follow it and in the same time meet each other criteria, like length.

**Comment by TomazErjavec** [comment]
I completely agree with @luhpeg, as exactly the same arguments hold for Slovenian ElTeC.
Maybe one way of keeping both views would be, rather than having eltec + eltec-extension,  to have eltec + eltec-balanced, where the latter could contain texts from the former, automatically selected to comply to the new balancing criteria, in the extent possible for individual languages.

**Comment by lb42** [comment]
We are not trying to build a corpus which includes only works by a (small number of) "canonical" authors. That is not a "new" criterion: it is clearly stated in all the original design documents. We are trying to build a corpus which represents, as far as possible, the full variety of novel production during the period. This means it should contain some "low quality texts" as you call them. The limitation on the number of texts per author is also not new, but I agree that it was not originally made explicit that we were thinking of a maximum. Again, the principle behind this is to maximize variety.

**Comment by CarolinOdebrecht** [comment]
I specified the titles per author criterion in the sampling document. If something there is not yet understandable, please let me know. 

The texts that are already digitized and encoded but do not meet these criteria (e.g. further texts of an author who already has three texts) can be stored in an extra folder in each language collection, see https://github.com/distantreading/WG1/issues/23
